projects/canopy/src/lib/button/docs/guide.stories.mdx

```diff
@@ -162,51 +162,51 @@ An important requirement is the need to add first and second attributes to the i
 
 ![Button component - what you should do](docs/button/do.png)
 
-1. **Do** make sure you choose a button that has sufficient contrast with the background
-2. **Do** use concise labels that are descriptive and help set the user's expectations
-3. **Do** distinguish between functional purposes
-4. **Do** use full-width buttons on mobile breakpoints
+1. **Do** make sure you choose a button that has sufficient contrast with the background.
+2. **Do** use concise labels that are descriptive and help set the user's expectations.
+3. **Do** distinguish between functional purposes.
+4. **Do** use full-width buttons on mobile breakpoints.
 
 ### Don't
 
 ![Button component - what you shouldn't do](docs/button/dont.png)
 
-1. **Don't** use a button variant that contrasts poorly with the background
-2. **Don't** use more than one primary button per page and don't use too many buttons in close proximity
-3. **Don't** wrap text labels
-4. **Don't** confuse priorities by using multiple primary CTA's at the same time
-5. **Don't** use overly wordy labels
-6. **Don't** centre align buttons
-7. **Don't** right align buttons
-8. **Don't** use uninformative labels
+1. **Don't** use a button variant that contrasts poorly with the background.
+2. **Don't** use more than one primary button per page and don't use too many buttons in close proximity.
+3. **Don't** wrap text labels.
+4. **Don't** confuse priorities by using multiple primary CTA's at the same time.
```

Comment:
```suggestion
4. **Don't** confuse priorities by using multiple primary CTAs at the same time.
```

Reply:
Oh @paul-request! I merged the PR - but feel free to add more comments and I will create a new PR to fix them 👍